### DISCUSSION TITLE
[SUGGESTION/BUG] Counterclockwise rotating rotators for reset position

### DISCUSSION THREAD

**Discussion by TIKIRobo** [tsunamayo/Starship-EVO#4037] [Suggestions]
### Short Description of the issue:
rotators will rotate in both directions, except when they need to
...

### Steps to reproduce the issue:
start rotating rotators from 359 to 360
...

### Environment
- Starship EVO version: 21w03a

### Additional information:

https://user-images.githubusercontent.com/56371294/105115462-16d35c00-5a86-11eb-8071-736a85382b8b.mp4

**Comment by Kaiser-Indrasil** [comment]
I believe Tsuna would need to implement the infinite rotators first for this to work.

**Comment by tsunamayo** [comment]
Yes it is by design for now. I want to add a few settings (many time you dont want it to loop, for a flip flop mechanism you want it to go back and forth - but first I want to do a rotator refactoring.
